### DISCUSSION TITLE
Can I remove a group from MDF file or modify the group timestamps?

### DISCUSSION THREAD

**Discussion by shangrilaer** [danielhrisca/asammdf#616] [Q&A] [answered]
I found the logger with wrong GPS group of MDF . 
1. iT put GPStime as timestamps but not the offset, this will make asammdf crazy with resampling.
2. I'm even crazy for the wrong GPstime as timestamps, this will be very big and make asammdf out of memory.

What I want do is to :
1. remove the GPS group directly before resampling to a dataframe.
2. or after dropping, I append the dataframe to the group? and save the mdf file back.

How Can I do this or Is there a easy way?

**Comment by JulienGrv** [comment, marked as answer]
I think there is the answer to your question  in issue #614
